extensions/binding/org.eclipse.smarthome.binding.ble.yeelightblue/ESH-INF/thing/yeelight_blue2.xml

```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="ble" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+	<!-- Sample Thing Type -->
+	<thing-type id="yeelight_blue2" listed="false">
+		<label>YeeLightBlue II Light Bulb</label>
```

Comment:
I expected a reference to the bridge (maybe not the bluegiga, but some generic ?, I am not sure ):
```
 <supported-bridge-type-refs>
      <bridge-type-ref id="...." />
 </supported-bridge-type-refs>

```